libraries/AP_AIS/AP_AIS.cpp

```diff
@@ -95,17 +95,44 @@ void AP_AIS::init()
     }
 
     _uart = AP::serialmanager().find_serial(AP_SerialManager::SerialProtocol_AIS, 0);
-    if (_uart == nullptr) {
-        return;
+    if (_uart != nullptr) {
+        _uart->begin(AP::serialmanager().find_baudrate(AP_SerialManager::SerialProtocol_AIS, 0));
     }
 
-    _uart->begin(AP::serialmanager().find_baudrate(AP_SerialManager::SerialProtocol_AIS, 0));
 }
```

Comment:
Why the re-oder here? Its functionally the same.

Reply:
true, but I just wanted it so that init looks like it can continue. The heart of a uart not being required

Reply:
@IamPete1 do you insist or can we keep this as I've changed it?

Reply:
nevermind.. whatever.. I'll change it back